Sources/MapboxNavigation/StatusView.swift

```diff
@@ -46,6 +46,39 @@ public class StatusView: UIControl {
         }
     }
     
+    var statuses: [Status] = []
+
+    /**
+    `Status` is a struct which stores information to be displayed by the `StatusView`
+     */
+    public struct Status {
+        public var identifier: String
+        let title: String
+        var spinner: Bool = false
+        let duration: TimeInterval
+        var animated: Bool = true
+        var interactive: Bool = false
+        var priority: Priority
+    }
+    
+    /**
+     `Priority` is used to display `Status`es by importance
+     */
+    public struct Priority: RawRepresentable {
+        public typealias RawValue = Int
```

Comment:
Since no specific values are defined, it would be simpler to define a type alias than a raw representable type alias inside a struct:

```
public typealias Priority = Int
```